src/openrct2/interface/Theme.cpp

```diff
@@ -165,6 +165,7 @@ WindowThemeDesc WindowThemeDescriptors[] =
     { THEME_WC(WC_EDITOR_SCENARIO_BOTTOM_TOOLBAR), STR_THEMES_WINDOW_BOTTOM_TOOLBAR_SCENARIO_EDITOR,       COLOURS_3(TRANSLUCENT(COLOUR_LIGHT_BROWN), TRANSLUCENT(COLOUR_LIGHT_BROWN), TRANSLUCENT(COLOUR_MOSS_GREEN)                     ) },
     { THEME_WC(WC_TITLE_EDITOR),                   STR_TITLE_EDITOR_TITLE,       COLOURS_3(COLOUR_GREY,                     COLOUR_OLIVE_GREEN,              COLOUR_OLIVE_GREEN                                 ) },
     { THEME_WC(WC_TILE_INSPECTOR),                 STR_TILE_INSPECTOR_TITLE,       COLOURS_2(COLOUR_LIGHT_BLUE,               COLOUR_LIGHT_BLUE                                                                   ) },
+    { THEME_WC(WC_VIEW_CLIPPING),                 STR_TILE_INSPECTOR_TITLE,       COLOURS_1(COLOUR_LIGHT_BLUE                ) },
```

Comment:
I finally clued in to how the colours work.  The icons in the top bar have the colour set in the background that is (should be) used in the window background.  For the view options this is COLOUR_DARK_GREEN.  Tabs in these windows are COLOUR_DARK_BROWN (as you suggested), but this window has no tabs.

I'll make the other changes suggested.